test/powershell/Modules/Microsoft.PowerShell.Utility/string.tests.ps1

```diff
@@ -14,7 +14,8 @@
             $pathWithoutWildcard = $TestDrive
             $pathWithWildcard = Join-Path $TestDrive '*'
 
-            $tempFile = New-TemporaryFile
+            # Here Get-ChildItem adds 'PSDrive' property
+            $tempFile = New-TemporaryFile | Get-ChildItem
```

Comment:
Although in this case, it'll have the same result, I think

```powershell
New-TemporaryFile | Get-Item
```

is more correct, particularly if other people use this pattern until we address `New-TemporaryFile`

Reply:
Fixed.